src/formats/PDB.cpp

```diff
@@ -199,6 +224,35 @@ void PDBFormat::read_CRYST1(Frame& frame, const std::string& line) {
     }
 }
 
+void PDBFormat::read_HELIX(const std::string& line) {
+    auto chain = line[19];
+    auto start = std::stoul(line.substr(21,4));
```

Comment:
Could you wrap this is a try block to provide a better error message in case of failure?